docs/index.md

```diff
@@ -197,14 +197,14 @@ your program. That means it's easy to process them afterwards.
 
 ```Lua
 -- You can round the animated health to whole numbers.
-local wholeHealth = Computed(function()
-	return math.round(health:get())
+local wholeHealth = Computed(function(use, scope)
+	return math.round(use(health))
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
local wholeHealth = Computed(function(use)
```

Reply:
Should be scoped
```suggestion
local wholeHealth = scope:Computed(function(use)
```

Reply:
done